packages/api/models/schemas.js

```diff
@@ -110,12 +110,6 @@ module.exports.collection = {
         + 'will be used as the granule ID.',
       type: 'string',
     },
-    reportToEms: {
```

Comment:
If we remove these from the API model schema, this will fail validation if a user has this explicitly set to false and tries to update  an unrelated field?

Is that a desirable behavior?

Reply:
Ah interesting. That does not sound very desirable. I noticed the `dataType` field's description says it's deprecated, so I'll follow suit with this.